src/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+
+const { randomDigitsGenerator } = require('./modules/randomDigitsGenerator');
+const { bullsCowsCalculator } = require('./modules/bullsCowsCalculator');
+const { handleInputOutput } = require('./modules/handleInputOutput');
+
+const bullsAndCowsGame = async() => {
+  const randomDigits = randomDigitsGenerator();
+
+  const userNumber = await handleInputOutput();
+
+  if (userNumber) {
+    const result = bullsCowsCalculator(randomDigits, userNumber);
+
+    if (result === 1) {
+      global.console.log(result);
+      global.console.log('You are winner !!!');
```

Comment:
after user win, game should be stoped

Reply:
the game stops after user won

Reply:
if I understood properly game ends when user guessed  4 bulls? Am I right?